backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -59,6 +59,12 @@ trait BackendLifecycleActorFactory {
   lazy val fileHashingActorCount: Int = 50
 
   def fileHashingActorProps: Props = FileHashingActor.props(fileHashingFunction)
+  
+  /*
+   * Returns credentials that can be used to authenticate to a docker registry server
+   * in order to obtain a docker hash.
+   */
+  def dockerHashCredentials(initializationDataOption: Option[BackendInitializationData]): Seq[Any] = Seq.empty
```

Comment:
It's `any` kind of credential object that the backend wants to pass along so the docker actor is able to make a proper request. It could be a generic `T` but the point is that each Docker flow can match against it and use it if they can. (Dockerhub flow recognizes `DockerCredentials`, GCR flow recognizes google `Credentials` etc...)
If there is a way to make it generic while still being able to pattern match it and avoid type erasure that would definitely be better but I couldn't find one.